crates/forge/tests/cli/zk_cmd.rs

```diff
@@ -122,3 +122,57 @@ contract CallEmptyCode is Test {
         .stdout_lossy()
         .contains("call may fail or behave unexpectedly due to empty code");
 });
+
+// Sending ETH to an EOA doesn't trigger empty code warning
+forgetest_async!(test_zk_can_send_eth_to_eoa, |prj, cmd| {
+    foundry_test_utils::util::initialize(prj.root());
+    prj.add_test(
+        "SendEthToEOA.t.sol",
+        r#"
+import "forge-std/Test.sol";
+
+contract SendEthToEOA is Test {
+    function testSendEthToEOA() external {
+        address eoa = makeAddr("Juan's Account");
+        vm.deal(address(this), 1 ether);
+        
+        (bool success,) = eoa.call{value: 1 ether}("");
+        assertTrue(success, "ETH transfer failed");
+    }
+}
+"#,
+    )
+    .unwrap();
+
+    cmd.args(["test", "--zksync", "--match-test", "testSendEthToEOA"]);
+    let output = cmd.assert_success().get_output().stdout_lossy();
+
+    assert!(!output.contains("call may fail or behave unexpectedly due to empty code"));
+});
+
+//Calling an address with Zero value should trigger empty code warning
+forgetest_async!(test_zk_can_call_empty_code_with_zero_value, |prj, cmd| {
```

Comment:
```suggestion
forgetest_async!(test_zk_calling_empty_code_with_zero_value_issues_warning, |prj, cmd| {
```
and delete the comment above